traffic_ops/v4-client/cachegroup_parameters.go

```diff
@@ -58,3 +58,10 @@ func (to *Session) CreateCacheGroupParameter(cacheGroupID, parameterID int, opts
 	reqInf, err := to.post(apiCachegroupParameters, opts, cacheGroupParameterReq, &data)
 	return data, reqInf, err
 }
+
+// CreateMultipleCacheGroupParameter associates multiple parameter with multiple Cache Group.
```

Comment:
nit: "parameter" and "Cache Group" should be plural: "Parameters" and "Cache Groups"

Reply:
Fixed.